application/src/main/java/org/thingsboard/server/service/ttl/AbstractTimeseriesCleanUpService.java

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright © 2016-2020 The Thingsboard Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.thingsboard.server.service.ttl;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.thingsboard.server.dao.util.PsqlTsAnyDao;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.SQLWarning;
+import java.sql.Statement;
+
+@PsqlTsAnyDao
+@Slf4j
+public abstract class AbstractTimeseriesCleanUpService {
+
+    @Value("${sql.ttl.ts_key_value_ttl}")
+    protected long systemTtl;
+
+    @Value("${sql.ttl.enabled}")
+    private boolean ttlTaskExecutionEnabled;
+
+    @Value("${spring.datasource.url}")
+    private String dbUrl;
+
+    @Value("${spring.datasource.username}")
+    private String dbUserName;
+
+    @Value("${spring.datasource.password}")
+    private String dbPassword;
+
+    @Scheduled(fixedDelayString = "${sql.ttl.execution_interval_ms}")
```

Comment:
Please add initial delay. @Scheduled(fixedDelayString = "${sql.ttl.execution_interval_ms}", initialDelayString = "${sql.ttl.execution_interval_ms}"). No need to overload the system at start.